multipaz-compose/src/commonMain/kotlin/org/multipaz/compose/Util.kt

```diff
@@ -1,5 +1,55 @@
 package org.multipaz.compose
 
 import androidx.compose.ui.graphics.ImageBitmap
+import org.multipaz.compose.camera.CameraFrame
 
 expect fun decodeImage(encodedData: ByteArray): ImageBitmap
+
+/**
+ * Extract an arbitrary geometry rectangular bitmap from the original bitmap.
+ *
+ * The method is used in [multipaz-mlkit] to crop and level the face image using MLKit face detection data
+ * as needed for further processing.
+ *
+ * **Usage Example**
+ *
+ *     // Extract the square shaped face bitmap around the center point between eyes and level it by eyes line.
+ *     val faceCenterX = (leftEye.position.x + rightEye.position.x) / 2
+ *     val faceCenterY = (leftEye.position.y + rightEye.position.y) / 2
+ *     val eyeOffsetX = leftEye.position.x - rightEye.position.x
+ *     val eyeOffsetY = leftEye.position.y - rightEye.position.y
+ *     val eyeDistance = sqrt(eyeOffsetX * eyeOffsetX + eyeOffsetY * eyeOffsetY)
+ *     val faceWidth = eyeDistance * 3 // Heuristic multiplier to fit the face normalized to the eyes pupilar distance.
+ *     val eyesAngleRad = atan2(eyeOffsetY, eyeOffsetX)
+ *     val eyesAngleDeg = eyesAngleRad * 180.0 / PI // Convert radians to degrees
+ *
+ *     // Call platform dependent bitmap transformation.
+ *     val croppedFaceImageBitmap =  cropRotateScaleImage(
+ *         frameData = frameData, // Platform-specific image data.
+ *         cx = faceCenterX.toDouble(), // Point between eyes
+ *         cy = faceCenterY.toDouble(), // Point between eyes
+ *         angleDegrees = eyesAngleDeg, // Eyes line rotation.
+ *         outputWidth = faceWidth.toInt(), // Expected face width for cropping *before* final scaling.
+ *         outputHeight = faceWidth.toInt(),// Expected face height for cropping *before* final scaling.
+ *         targetWidth = 256 // Final square image size (for database saving and face matching tasks).
+ *     )
```

Comment:
Enclose the code sample using three back-ticks, like other places with sample code (See e.g. `X500Name.fromName()`).